app/views/admin/notes/_form.html.erb

```diff
@@ -1,5 +1,9 @@
 <%= form_with model: [:admin, note], local: true do |f| %>
   <%= f.hidden_field :submission_id %>
   <%= f.text_area :body, required: "required" %>
-  <%= f.submit 'Create' %>
-<% end %>
\ No newline at end of file
+  <%= f.label :add_note_to_user do %>
+    <%= f.check_box :add_note_to_user %>
+    Add a note to the user
```

Comment:
@bhoggard what do you think would be a descriptive description here? Do you think something like this is enough to understand what this checkbox means?